src/components/Modals/GuideModal/GuideModal.tsx

```diff
@@ -0,0 +1,41 @@
+import { useHistory } from 'react-router-dom';
+import { GUIDE_URL } from '@/constants';
+import { Localize } from '@deriv-com/translations';
+import { Button, Modal, Text } from '@deriv-com/ui';
+import './GuideModal.scss';
+
+type TGuideModalProps = {
+    isModalOpen: boolean;
+    onRequestClose: () => void;
+};
+
+const GuideModal = ({ isModalOpen, onRequestClose }: TGuideModalProps) => {
+    const history = useHistory();
+
+    const onGetStarted = () => {
+        history.push(GUIDE_URL);
+        onRequestClose();
+    };
+
+    return (
+        <Modal className='guide-modal' isOpen={isModalOpen} shouldCloseOnOverlayClick={false}>
```

Comment:
related to what I had said above, no need to add another scss file since its so small
```suggestion
        <Modal className='w-[44rem]' isOpen={isModalOpen} shouldCloseOnOverlayClick={false}>
```